e2e_test/error_ui/simple/main.slt

```diff
@@ -8,18 +8,6 @@ Caused by:
 Near "selet"
 
 
-statement error
-create function int_42() returns int as int_42 using link 'localhost:8815';
-----
-db error: ERROR: Failed to run the query
-
-Caused by these errors (recent errors listed first):
-  1: failed to connect to UDF service
-  2: transport error
-  3: error trying to connect
-  4: invalid URL, scheme is missing
-
```

Comment:
Why remove this test?

Reply:
Oh because the URL is now valid.

Reply:
Would you please replace it with a new test case instead, like invalid `<host>:<port>` format.

Reply:
Added two test cases.